package.json

```diff
@@ -44,52 +46,687 @@
+          },
+          {
+          "key":"TWD",
+          "value":"TWD - NT$"
+          },
+          {
+          "key":"TZS",
+          "value":"TZS - TSh"
+          },
+          {
+          "key":"UAH",
+          "value":"UAH - грн."
+          },
+          {
+          "key":"UGX",
+          "value":"UGX - USh"
+          },
+          {
+          "key":"USD",
+          "value":"USD - $"
+          },
+          {
+          "key":"UYU",
+          "value":"UYU - $U"
+          },
+          {
+          "key":"UZS",
+          "value":"UZS - сўм"
+          },
+          {
+          "key":"VND",
+          "value":"VND - ₫"
+          },
+          {
+          "key":"VUV",
+          "value":"VUV - VT"
+          },
+          {
+          "key":"WST",
+          "value":"WST - WS$"
+          },
+          {
+          "key":"XAF",
+          "value":"XAF - F"
+          },
+          {
+          "key":"XCD",
+          "value":"XCD - $"
+          },
+          {
+          "key":"XOF",
+          "value":"XOF - F"
+          },
+          {
+          "key":"XPF",
+          "value":"XPF - F"
+          },
+          {
+          "key":"YER",
+          "value":"YER"
+          },
+          {
+          "key":"ZAR",
+          "value":"ZAR - R"
+          },
+          {
+          "key":"ZMW",
+          "value":"ZMW - ZK"
+          }
+        ]
       },
       {
-        "key": "DESTINATION_PLACE",
+        "key": "originPlace",
+        "label": "Choose an origin place for your flight.",
+        "help": "",
+        "required": true,
+        "order": 1,
+        "value": "AUS",
+        "valueLabel": "Austin Bergstrom International Airport",
+        "controlType": "search",
+        "dynamic": true,
+        "options": []
+      },
+      {
+        "key": "destinationPlace",
         "label": "Choose an destination place.",
         "help": "",
         "required": true,
         "order": 1,
-        "placeholder": "ABC",
-        "hint": "The place must have this format: ABC",
-        "controlType": "textbox"
+        "value": "SFO",
+        "valueLabel": "San Francisco International Airport",
+        "controlType": "search",
+        "dynamic": true,
+        "options": []
       },
       {
-        "key": "DEPART_DATE",
+        "key": "departDate",
         "label": "Choose an depart date.",
         "help": "",
         "required": true,
         "order": 1,
         "placeholder": "yyyy-mm-dd",
-        "hint": "The date must have this format: yyyy-mm-dd",
         "controlType": "textbox"
       },
       {
-        "key": "RETURN_DATE",
+        "key": "returnDate",
         "label": "Choose an optional return date.",
         "help": "Optional",
         "required": false,
         "order": 1,
         "placeholder": "yyyy-mm-dd",
-        "hint": "The date must have this format: yyyy-mm-dd",
         "controlType": "textbox"
+      },
+      {
+        "key": "threshold",
```

Comment:
Should not be called threshold. Variable name should be more explicit. E.g: maxAffordablePrice.